openlibrary/templates/books/custom_carousel.html

```diff
@@ -42,6 +44,7 @@
           <img class="bookcover" loading="lazy"
             width="130" height="200"
             title="$('%s%s'%(title,byline))"
+            $src_attr="$(data_url)"
```

Comment:
why the need for src_attr here? Why not just src="$(data_url)" ?

Reply:
@jdlrobson If I write just src validation will give an error that I have 2 src